sceval_frontend/src/utils/Constants.ts

```diff
@@ -0,0 +1,21 @@
+
+export class CONSTANTS {
```

Comment:
Ditto

Reply:
@reprimande, thanks for the feedback. I changed the class name and also changed it from class to namespace.